source/statements.tex

```diff
@@ -843,10 +843,10 @@
 \end{note}
 \begin{note}
 A \tcode{return} statement can involve
-an invocation of a constructor to perform a copy or move of the operand
+a copy or move of the operand
 if it is not a prvalue or if its type differs from the return type of the function.
-A copy operation associated with a \tcode{return} statement can be elided or
-converted to a move operation if an automatic storage duration variable is returned\iref{class.copy.elision}.
+The copy operation associated with a \tcode{return} statement can be elided or
```

Comment:
Why "the" copy operation? There might be none.

Reply:
My thought process was: There _might_ be none, but if there are any, then there is _one_. "A copy operation" reads to me as if there might be several, and any-or-all of them might be converted. But what we mean is that there is at most one copy, and, if _the_ copy exists, then _the_ copy can be elided.

Still, I can see the logic either way, and `s/A/The/` isn't important to this patch, so I've reverted to `A`.